### PR TITLE
fix(tooltip): change reference element

### PR DESCRIPTION
**Pull request by larsdouweschuitema**
BREAKING CHANGE: element changed from button to div

### Why this change?
Duplicate button when using your own component inside the slot.
Popper requires a slot named reference.
Not only will it render a duplicate button in the DOM, it also doesnt allow you to trigger the tooltip on button click due to duplicate elements.

https://www.npmjs.com/package/vue-popperjs

### CODE REVIEW

File: src/components/ui-tooltip.vue [hunk 1]
```diff
@@ -9,9 +9,9 @@
                 <slot name="content" />
             </div>
 
-            <button slot="reference" class="focus:outline-none">
+            <div slot="reference">
```
Comment (dine) [inline src/components/ui-tooltip.vue:12]: Do you think we should add a `class="cursor-pointer"` here now that the button is gone?

**Comment by larsdouweschuitema** [comment]
I also noticed hover is not working, but thats regardless of using button or div. We should check this out in another issue.

**Review by dine — APPROVED** [review]

**Comment by dine** [comment]
> I also noticed hover is not working, but thats regardless of using button or div. We should check this out in another issue.

I notice it doesn't work on the Canvas tab, but it does on Docs https://funda-frontend.github.io/ui/latest/?path=/docs/visual-tooltip--tooltip

File: src/components/ui-tooltip.vue (hunk 1 above)
Comment (larsdouweschuitema) [reply src/components/ui-tooltip.vue:12]: @dine If you for example use your own component e.g. UiButton it will have the cursor pointer on there.

**Comment by larsdouweschuitema** [comment]
> > I also noticed hover is not working, but thats regardless of using button or div. We should check this out in another issue.
> 
> I notice it doesn't work on the Canvas tab, but it does on Docs https://funda-frontend.github.io/ui/latest/?path=/docs/visual-tooltip--tooltip

Interesting. I see the hover working also on the doc. Maybe its just a storybook issue?

**Comment by ConnorChristensen** [comment]
I think this has the unintended side effect of no longer having a pointer on the reference, and the text can be highlighted.

**Pull request merged**